pkg/option/config.go

```diff
@@ -2691,6 +2706,24 @@ func (c *DaemonConfig) populateNodePortRange() error {
 	return nil
 }
 
+func (c *DaemonConfig) populateNodePortAddresses() error {
+	nodePortAddresses := viper.GetStringSlice(NodePortAddresses)
+
+	for _, ip := range nodePortAddresses {
+		_, cidr, err := net.ParseCIDR(ip)
+		if err != nil {
+			return fmt.Errorf("Parsing subnet, %q, for %s: %v", ip, NodePortAddresses, nodePortAddresses)
```

Comment:
Nit: generally error strings start lower-cased.

Reply:
I think an error string of `invalid subnet ... ` reads better as an error, rather than starting with `parsing`, because it sounds like the start of a log msg.

Reply:
`pkg/option` is horribly inconsistent in this regard. I was copying what I was seeing, but I agree with you that errors shouldn't be upper case (or even sentences for that matter).